definitions/v4/drunkenslug.yml

```diff
@@ -0,0 +1,164 @@
+---
+id: drunkenslug-yml
+name: DrunkenSlug (YML)
```

Comment:
Would prefer to keep the names clean without `(YML)` if we can. It means nothing to the average user